xtask/src/utils.rs

```diff
@@ -30,16 +28,16 @@ pub fn ask_to_run(mut cmd: Command, ask: bool, reason: &str) -> Result<(), DynEr
         io::stdin().read_line(&mut buf).unwrap();
         match buf.trim().to_lowercase().as_ref() {
             "" | "y" | "yes" => {}
-            "n" | "no" => return Err("Aborting as per your request".into()),
-            a => return Err(format!("Invalid answer `{a}`").into()),
+            "n" | "no" => anyhow::bail!("Aborting as per your request"),
+            a => anyhow::bail!(format!("Invalid answer `{a}`")),
         };
     } else {
-        eprintln!("Running `{cmd:?}` to {reason}.");
+        log::info!("Running `{cmd:?}` to {reason}.");
```

Comment:
Don't know if this makes sense. xtask is supposed to be just like a shell script, but in Rust. The fewer bells and whistles it has, the better.

Reply:
you are right removed that and replaced it with println